client/src/components/VideoBackground/VideoBackground.css

```diff
@@ -0,0 +1,11 @@
+video.fullscreen { 
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    min-width: 100%;
+    min-height: 100%;
+    width: auto;
+    height: auto;
+    z-index: -100;
+    transform: translate(-50%, -50%);
+}
```

Comment:
We don't need this anymore, please delete.

Reply:
Looks like we can delete all files created for the video component since we're not using it anymore.